language/src/data/languages/language_fa_IR.ycp

```diff
@@ -0,0 +1,43 @@
+/**
+ * This file contains the description of the language needed by yast2-country
+ * module.
+ *
+ * Translation of the "language name" to actual YaST language be must applied
+ * each time this file is invoked.
+ *
+ * If you want to add new language to YaST using a file of this type provided
+ * by some module different from yast2-country (e.g. in some Add-On Product),
+ * consider using special textdomain, so it is possible to provide also
+ * translations of the 'language name' string to all installed languages.
+ *
+ * Author:	Jiří Suchomel <jsuchome@suse.cz>
+ * $Id$
+ */
```

Comment:
Please, remove the "Author:" and the "$Id$" part. Those are leftover from the old way of doing things in CVS times.

Even more, in general I find repeating the whole header in every language file to be a bit too much.